website/content/docs/concepts/client-count/client-usage.mdx

```diff
@@ -147,15 +147,19 @@ $ curl --header "X-Vault-Token: <TOKEN>" \
   [documentation](/vault/docs/concepts/client-count#entity-assignment-with-acme)
   for more details.
 
-    ![ACME clients example](/img/ui-usage-metrics-acme.png)
+    ![ACME clients example](/img/ui-client-usage-acme.png)
```

Comment:
"ACME clients example" is not useful alt text. Why is this screenshot useful? What does it highlight or demonstrate?